src/MCPClient/lib/clientScripts/elasticSearchIndexProcessTransfer.py

```diff
@@ -30,22 +30,14 @@
 from custom_handlers import get_script_logger
 import elasticSearchFunctions
 
+from django.conf import settings as mcpclient_settings
 
 logger = get_script_logger('archivematica.mcp.client.elasticSearchIndexProcessTransfer')
 
 
 if __name__ == '__main__':
-    config_file = '/etc/archivematica/MCPClient/clientConfig.conf'
-    config = ConfigParser.SafeConfigParser()
-    config.read(config_file)
-
-    elasticsearchDisabled = False
-    try:
-        elasticsearchDisabled = config.getboolean('MCPClient', "disableElasticsearchIndexing")
-    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError):
-        pass
-    if elasticsearchDisabled is True:
-        logger.info('Skipping indexing: indexing is currently disabled in %s.', config_file)
+    if mcpclient_settings.DISABLE_SEARCH_INDEXING is True:
```

Comment:
`DISABLE_SEARCH_INDEXING` is defined as a boolean so that's the type of value expected.